### PR TITLE
Fixes #17874: Add support of Ubuntu 20 in doc

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/17874

### CODE REVIEW

File: src/reference/modules/installation/pages/operating_systems.adoc [hunk 1]
```diff
@@ -74,7 +74,7 @@ The following operating systems are supported as a root or relay server:
 | OS | Version | Architecture
 
 | Debian | 9 and 10 | 64bit
-| Ubuntu | 16.04 and 18.04 LTS | 64bit
+| Ubuntu | 16.04 to 20.04 LTS | 64bit
```
Comment (amousset) [inline src/reference/modules/installation/pages/operating_systems.adoc:77]:
```suggestion
| Ubuntu | 16.04 and 20.04 LTS | 64bit
```

File: src/reference/modules/installation/pages/operating_systems.adoc [hunk 2]
```diff
@@ -40,7 +40,7 @@ available for these platforms as part of the https://www.rudder.io/en/pricing/su
 | 8 | 32bit, 64bit, armhf (Raspbian)
 | 9 | 32bit, 64bit, arm64, armhf (Raspbian)
 .2+| Ubuntu | 10.04 to 16.04 LTS | 32bit, 64bit
-| 18.04 LTS | 64bit
+| 18.04 to 20.04 LTS LTS | 64bit
```
Comment (amousset) [inline src/reference/modules/installation/pages/operating_systems.adoc:43]:
```suggestion
| 18.04 and 20.04 LTS | 64bit
```

File: src/reference/modules/installation/pages/operating_systems.adoc (hunk 1 above)
Comment (VinceMacBuche) [inline src/reference/modules/installation/pages/operating_systems.adoc:77]: Server is only 18 and 20 on 6.1!!

File: src/reference/modules/installation/pages/operating_systems.adoc [hunk 3]
```diff
@@ -19,7 +19,7 @@ available freely for these platforms:
 
 | Debian | 8, 9 and 10 | 32bit, 64bit
 .2+| Ubuntu | 14.04 and 16.04 LTS | 32bit, 64bit
-| 18.04 LTS | 64bit
+| 18.04 LTS to 20.04 LTS | 64bit
```
Comment (amousset) [inline src/reference/modules/installation/pages/operating_systems.adoc:22]:
```suggestion
| 18.04 LTS and 20.04 LTS | 64bit
```

File: src/reference/modules/installation/pages/operating_systems.adoc (hunk 1 above)
Comment (peckpeck) [reply src/reference/modules/installation/pages/operating_systems.adoc]: yes

**Comment by VinceMacBuche** [comment]
You need to rebase
 there is 5 commits

**Comment by ElaadF** [comment]
PR rebased

**Comment by VinceMacBuche** [comment]
but you lost all changes when rebasing @ElaadF

**Comment by ElaadF** [comment]
Commit modified

**Pull request merged**